README.md

```diff
@@ -104,7 +104,7 @@ The Operations Team takes care of legal documents, invoicing and remittances.
 
 ## :pencil: Process
 
-> **:loudspeaker:** We ask teams to follow the process below, which has **faster processing times** and provides grants of up to **$30,000 for initial applications** and **$100,000 for follow-up** ones disbursed in Bitcoin, USDT or DAI. However, if you want to apply in **private**, need to be paid out in **fiat** and/or you **seek funding of $100,000+** and your work cannot be split into smaller milestones, you can apply [:arrow_right: here](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform).
+> **:loudspeaker:** Regular grants are **public** and provide grants of up to **$30,000 for initial applications** and **$100,000 for follow-up** ones, disbursed in **Bitcoin, USDT or DAI**. If you want to apply in **private**, need to be paid out in **fiat** and/or you **seek funding of $100,000+** and your work cannot be split into smaller milestones, you can apply [:arrow_right: here](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform).
```

Comment:
Sorry, I didn't see this. My suggestion on the other PR conflicts with it. Also not a fan since there's no distinction between "regular" and "irregular" grants elsewhere, and also "grants provide grants" ;)

Reply:
Good points. I, however, am don't particularly love `strongly encourage`. But I didn't see your suggestion on the other PR until just now, so I'll close this one and make a suggestion on your review.